README.md

```diff
@@ -1,9 +1,39 @@
 # TypeOfNaN JavaScript Quizzes
 
+![TypeOfNaN Logo javascript quizzes](typeOfNaN-logo.jpg)
+
 Learn JavaScript fundamentals through fun and challenging quizzes!
 
 View the app online here: [https://quiz.typeofnan.dev](https://quiz.typeofnan.dev)
 
+# How to run the app locally
+
+In order to use this app locally, the package manager _yarn_ needs to be installed
+
+If you don't have it installed yet, head over to:
+[https://yarnpkg.com/en/docs/install](https://yarnpkg.com/en/docs/install)
+and install the latest yarn version for your system.
+
+### 1. Clone the repo
+
+Run this command to clone the repo
+
+`git clone https://github.com/nas5w/typeofnan-javascript-quizzes`
+
+### 2. Install dependencies
+
+The first before you can use the app, you have run this command to install all the dependencies
```

Comment:
Remove "The" from beginning of line. This line should be:

First, before you can use the app, you have run this command to install all the dependencies